devtools/conda-recipe/README.md

```diff
@@ -6,7 +6,7 @@ it, running the tests, and then if successful pushing the package to binstar
 (and the docs to AWS S3). The binstar auth token is an encrypted environment
 variable generated using:
 
-binstar auth -n repex-travis -o omnia --max-age 22896000 -c --scopes api:write
+binstar auth -n repex-travis -o conda-forge --max-age 22896000 -c --scopes api:write
```

Comment:
I don't think we're using `binstar` anymore. 

Does this repo need to be updated to the current [MolSSI cookiecutter?](https://github.com/MolSSI/cookiecutter-cms)

Reply:
@Lnaden any thoughts on binstar or updating the cookiecutter?